aten/src/ATen/native/SharedReduceOps.h

```diff
@@ -114,7 +124,12 @@ struct MeanOps {
     return reduce(a, b);
   }
 
-  inline C10_DEVICE acc_t project(acc_t a) const {
+  inline C10_DEVICE acc_t project(acc_t a, int value = 0) const {
+#if defined(__CUDACC__) || defined(__HIPCC__)
+    assert(value == 0);
```

Comment:
Because I added result_index to the signature I expect that this index will not exceed expected results count. In this case for Mean we do not expect more that one result, so index should be 0.